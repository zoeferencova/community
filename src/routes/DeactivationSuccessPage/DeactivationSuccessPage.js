import React from "react";
import { useNavigate } from "react-router-dom";
import { ButtonDark } from "../../components/Utils/Utils";

import styles from "./DeactivationSuccessPage.module.css"

const DeactivationSuccessPage = () => {
    const navigate = useNavigate();
    return (
        <div className={styles.container}>
            <h3>Your account has been deactivated</h3>
            <ButtonDark onClick={() => navigate("/")}>Go to Home Page</ButtonDark>
        </div>
    )
}

export default DeactivationSuccessPage;