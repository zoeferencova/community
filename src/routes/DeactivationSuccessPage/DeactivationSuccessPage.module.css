.container {
    margin-top: 200px;
    display: flex;
    flex-direction: column;
    align-items: center;
}

.container h3 {
    color: rgb(0, 0, 0);
    font-size: 18px;
    margin-bottom: 35px;
    font-weight: 500;
}