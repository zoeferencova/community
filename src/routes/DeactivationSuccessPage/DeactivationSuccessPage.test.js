import React from "react";
import { render } from "enzyme";
import DeactivationSuccessPage from "./DeactivationSuccessPage";
import { BrowserRouter } from "react-router-dom";
import CommUnityContext from "../../contexts/context";

import mockContext from "../../contexts/mock-context";

describe("DeactivationSuccessPage", () => {
    it("renders DeactivationSuccessPage component given mock context data", () => {
        const wrapper = render(
            <CommUnityContext.Provider value={mockContext}>
                <BrowserRouter>
                    <DeactivationSuccessPage />
                </BrowserRouter>
            </CommUnityContext.Provider>
        );
        
        expect(wrapper).toMatchSnapshot();
    });
});