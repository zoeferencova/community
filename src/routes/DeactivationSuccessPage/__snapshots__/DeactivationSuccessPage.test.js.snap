// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DeactivationSuccessPage renders DeactivationSuccessPage component given mock context data 1`] = `
<div
  class="main"
>
  <h3>
    Your account has been successfully deactivated
  </h3>
  <button
    class="buttonDark "
  >
    Go to Home Page
  </button>
</div>
`;
